### PR TITLE
I2321 - remove user/group reading permissions from users

### PR DESCRIPTION
**Pull request by littledice**

### CODE REVIEW

File: migrations/sql/V2018.10.30.1809__UpdateRolesAndPermissions.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+DELETE FROM role_permission
+WHERE role = 1
+AND permission IN ('estimates.read',
```
Comment (littledice) [inline migrations/sql/V2018.10.30.1809__UpdateRolesAndPermissions.sql]: noticed that all users have `estimates.read` which also seems wrong

File: migrations/sql/V2018.10.30.1809__UpdateRolesAndPermissions.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
+DO $$
+  declare admin int;
+  declare developer int;
+  declare member int;
+begin
+  SELECT id INTO admin FROM role WHERE name = 'admin';
+  SELECT id INTO developer FROM role WHERE name = 'developer';
+  SELECT id INTO member FROM role WHERE name = 'member';
```
Comment (richfitz) [inline migrations/sql/V2018.10.30.1809__UpdateRolesAndPermissions.sql]: does the `modelling-group` scope here interact with this at all? I must admit I've fallen behind on understanding the auth model and if you go through it with Emma I'd appreciate a refresher

**Comment by richfitz** [comment]
This looks good - it seems there might be other roles that could use this but I think it's safer to add them later as we find that out - shall I merge it?

**Comment by littledice** [comment]
Yeah thanks. I've opened a ticket to audit all roles, I think they could do with a bit of re-thinking as a lot were added before we really knew how people would end up interacting with Montagu https://vimc.myjetbrains.com/youtrack/issue/VIMC-2324

**Pull request merged**